chapters/getting_started_linux.md

```diff
@@ -0,0 +1,308 @@
+# Getting started with OpenCL on Ubuntu Linux
+
+OpenCL is included in all the main distributions of Linux. In this guide we choose Ubuntu as one of the most widespread variants, but all of the distributives have some sort of centralized system of package maintaining and OpenCL realizations can be installed from there.
+
+In this guide we're going to use the following tools:
+
+- C/C++ compiler (gcc & Clang)
+- CMake (Cross-platform Make)
+- Git
+- Vcpkg (Cross-platform pacakge management)
+- The command-line
+- Visual Studio Code
+
+Steps will be provided to obtain a minimal and productive environment.
+
+## Installation
+
+In this guide we'll be using latest (at the time of writing) Ubuntu 20.04 LTS. Installation for the most part will happen via [apt (Advanced Packaging Tool)](https://en.wikipedia.org/wiki/APT_(software)), the definitive command-line tool for installing software on Debian-base Linux distributives. It is installed with the system automatically.
+
+_(NOTE: installation commands should be issued with root access, so that use of privilege-escalation by `sudo` is mandatory.)_
+
+First update your system. Open your favorite shell in your favorite terminal and run:
+
+``` bash
+sudo apt update
+sudo apt upgrade
+```
+
+### C/C++ compiler
+
+Open your favorite shell in your favorite terminal and run:
+
+``` bash
+sudo apt install build-essential -y
+```
+
+This will install canonical GNU compiler set `g++/GNU 9.3`, debugger `gdb`, as well as some tools (e.g. `make`, `binutils`) and libraries for compilation and linking of programs.
+
+Another popular choice is the usage of `Clang` compiler, based on intermediate representation of the C-like languages on `LLVM` (originally Low Level Virtual Machine). To use it open your favorite shell in your favorite terminal and run:
+
+``` bash
+sudo apt install clang -y
+```
+
+This will install `Clang 10` and `LLVM 10`.
+
+### Git
+
+You most likely already have Git installed. If not, we'll install it, because this is what is needed to keep your repositories up-to-date. Open your favorite shell in your favorite terminal and run:
+
+``` bash
+sudo apt install git -y
+```
+
+### CMake
+
+If you do not have CMake installed yet, we'll install it, as it's the primary supported build system, and it will make our builds much simpler. Open your favorite shell in your favorite terminal and run:
+
+``` bash
+sudo apt install cmake ninja-build -y
+```
+
+### Visual Studio Code
+
+While IDEs are highly opinionated, for an IDE with widespared adoption and small footprint, VS Code has everything (and more) to get off the ground. It should be installed differently, though. The first way is to use GUI of `Ubuntu Software`, where you can search for `code` and install it. Whether you are a fan of command-line tools, use the second method: open your favorite shell in your favorite terminal and run
```

Comment:
The English of this paragraph needs some help. Here are my suggestions:
```suggestion
While IDEs are highly opinionated, for an IDE with widespread adoption and small footprint, VS Code has everything (and more) to get off the ground. It can be installed in two ways. The first way is to use the `Ubuntu Software` GUI, where you can search for `code` and install it in one click. If you are a fan of command-line tools, use the second method: open your favorite shell in your favorite terminal and run
```